NT-Line-team/Behave_test
# tamhom-issue53: Implement codes for printing invoices

Excercise #53

## Overview

An invoice item has the following main attributes: name, quantity (qty), unit price and amount. Write a Python program that lets user input a list of items. When user finishes data inputting process, the program should print out the invoice as below:
```
Item	Name	Qty	Unit Price	Amount
1	Apple iPhone	2	650	1300
2	Samsung TV	5	300	1500
3	Sony Washing Machine	1	750	750
Amount			3550
VAT (10%)			355
TOTAL AMOUNT			3905
```
## Verification Steps

1. Copy file to IDE Pycharm and run or run with command:
python ./printing_invoices.py
2. Enter data to verify as below.
```
Name	Qty	Unit Price	Amount
Sony Washing Machine	1	750	750
Apple iPhone	2	650	1300
Samsung TV	5	300	1500
Amount			3550
VAT (10%)			355
TOTAL AMOUNT			3905
```
And the result shows as:
```
Name	Qty	Unit Price	Amount
Apple iPhone	2	650	1300
Samsung TV	5	300	1500
Sony Washing Machine	1	750	750
Amount			3550
VAT (10%)			355
TOTAL AMOUNT			3905
```
